web/src/components/datasets/DatasetInfo.tsx

```diff
@@ -39,6 +42,23 @@ type DatasetInfoProps = {
 } & JobFacetsProps &
   DispatchProps
 
+const formatColumnTags = (tags: string[]) => {
+  const theme = createTheme(useTheme())
+  return <>{tags.map((tag, index) => (
+    <span
```

Comment:
I don't think we need the span here. I think we can just use the chip itself with marginRight applied to that component itself.

For applying `marginRIght` I would say something like:
marginRight: `index < tags.length - 1 ? theme.spacing(1) : 0`

Reply:
Cool - away racing this weekend but will fix it up Monday (NZ time)